### PR TITLE
fix(gwt editor): fix css in gwt editor

### PR DESCRIPTION
**Pull request by aeng**
https://zanata.atlassian.net/browse/ZNTA-1140

---
This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-server/1189)

### CODE REVIEW

**Comment by aeng** [comment]
@zanata-jenkins retest this please

**Comment by huangp** [comment]
:lgtm:

<details><summary><i>Previously, aeng (Alex Eng) wrote…</i></summary>
> @zanata-jenkins retest this please
</details>


---

Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-server/1189#-:-KJI_bodRBLnaJnQg6rT:1417088721)*

**Comment by djansen-redhat** [comment]
@aeng this does not seem to work for me

File: zanata-war/src/main/resources/org/zanata/webtrans/public/Application.xhtml [hunk 1]
```diff
@@ -68,8 +68,8 @@
 
     <div class='Ai(st) D(f) Fld(c) Flxs(0)'>
       <div class="Ai(st) D(f) Fld(c) Fld(r)--sm Flxs(0) H(100vh)">
-        <div id="root" class="H(100%)" />
-        <div id="container" class="Ovx(s)"/>
+        <div id="root" class="Or(1) Or(0)--sm" />
```
Comment (djansen-redhat) [inline zanata-war/src/main/resources/org/zanata/webtrans/public/Application.xhtml]: Should there be a H(100%) at the end of this class?

**Comment by djansen-redhat** [comment]
*[zanata-war/src/main/resources/org/zanata/webtrans/public/Application.xhtml, line 71 \[r2\]](https://reviewable.io:443/reviews/zanata/zanata-server/1189#-KJEmPcAw0dmOgd-JAaI-r2-71:-KJYQESdelOF2NXKy-rk:-1090532373) ([raw file](https://github.com/zanata/zanata-server/blob/efb9f2094b1c07310272856eee9d21d432b40708/zanata-war/src/main/resources/org/zanata/webtrans/public/Application.xhtml#L71)):*
<details><summary><i>Previously, djansen-redhat (Damian Jansen) wrote…</i></summary>
> Should there be a H(100%) at the end of this class?
</details>
OK - a bug in Chrome 50.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-server/1189)*

**Comment by djansen-redhat** [comment]
Reviewed 2 of 3 files at r1, 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-server/1189)*

**Comment by djansen-redhat** [comment]
:white_check_mark: Tested. This did take some time to verify, due to an apparent bug in Chrome version 50.x.

**Pull request merged**